frameworks/PHP/workerman/benchmark_config.json

```diff
@@ -44,6 +44,25 @@
       "display_name": "workerman postgres",
       "notes": "",
       "versus": "php"
+    },
+    "async": {
+      "db_url": "/db",
+      "fortune_url": "/fortune",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
```

Comment:
This does not look correct to me - `platform` classification is for barebones infrastructure for servicing HTTP requests, but does not include a framework at all. Given what the source code looks like, I would say the classification of `workerman` is either `micro` or `fullstack` depending on what functionality is provided.

I apologize, it appears that our definitions for these have been removed from the website and not yet added to the wiki - I will get them in there.

Reply:
I think that is a platform. The code only is for servicing HTTP request.
But using php in cli mode, whitout any webserver, fastcgi, ...
It's like nodejs but in php.

Reply:
And the definitions are in : https://www.techempower.com/benchmarks/#section=motivation

Reply:
Basically, yeah.